clowdapp.yaml

```diff
@@ -444,7 +444,7 @@ parameters:
 - name: CPU_REQUEST_PROM
   value: 100m
 - name: MEMORY_LIMIT_SERV
-  value: 800Mi
+  value: 1000Mi
```

Comment:
for a 1Gi (1000Mi) limit, request at least 500Mi to keep the Limit/request ratio happy.

```
10:53:43   - ReplicaFailure: True (pods "compliance-service-6fd5cbf6cb-8z2k6" is forbidden: memory max limit to request ratio per Pod is 2, but provided ratio is 2.400000)
```